### DISCUSSION TITLE
Question: Why did you choose the name ninjamuffin99?

### DISCUSSION THREAD

**Discussion by aflacc** [FunkinCrew/Funkin#1563] [Q&A]
how 2 delete

**Comment by ghost** [comment]
I just kinda thought of something and it stuck. Made a name for the freenode IRC network

**Comment by Windows1Door2Daniel** [comment]
Maybe just randomly

like my name

i thinked a door with windows OS colors

**Comment by libauror-a** [comment]
i just thaught of a plank in my dream snd i changed to that
